CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
-1.0.0-preview3 (June 17, 2020)
-==============================
+# 1.0.0-preview4 (Ongoing)
+
+## New Features
+
+* ### `LowAudioLevel` detection and warning in `InputTest`.
+
+  During the runtime of the test, if low audio levels are detected from the captured input device, the test will emit a `warning` event. The test will emit a `warning-clear` event when nominal audio levels are detected only after the `low-audio-level` `warning` has been fired.
```

Comment:
Is this `warning-cleared` or `warning-clear`?

Reply:
Good catch, to keep consistency with `twilio-client.js` we should use `warning-cleared`.